packages/openid4vc/src/openid4vc-issuer/handler/credentialEndpointHandler.ts

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright 2025 Velocity Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
```

Comment:
I know this is not important at this stage of the PR, but just as an FYI: 
I would rather not add these headers to all files. IF you want copyright credited, it can be added to the main license file (see example from Ontario: https://github.com/openwallet-foundation/credo-ts/blob/main/LICENSE#L190)